Tasks/InstallAppleProvisioningProfile/task.json

```diff
@@ -1,7 +1,7 @@
 {
     "id": "0f9f66ca-250e-40fd-9678-309bcd439d5e",
     "name": "InstallAppleProvisioningProfile",
-    "friendlyName": "Install Apple Provisioning Profile",
+    "friendlyName": "Install Apple ProvProfile (Preview)",
```

Comment:
To fit under 40 characters in preview, how about "Install Apple Profile (Preview)"?